caf-logging-tomcat-juli/pom.xml

```diff
@@ -0,0 +1,217 @@
+                                    <shadedPattern>org.apache.juli.logging.org.slf4j</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>ch.qos.logback</pattern>
+                                    <shadedPattern>org.apache.juli.logging.ch.qos.logback</shadedPattern>
+                                </relocation>
+                               
+                                <!-- Located in org.apache.juli.logging.ch.qos.logback.classic.util -->
+                                <relocation>
+                                    <pattern>logback.configurationFile</pattern>
+                                    <shadedPattern>juli-logback.configurationFile</shadedPattern>
+                                </relocation>
+                                <!-- Located in org.apache.juli.logging.org.slf4j.MarkerFactory -->
+                                <relocation>
+                                    <pattern>logback.ContextSelector</pattern>
+                                    <shadedPattern>juli-logback.ContextSelector</shadedPattern>
+                                </relocation>
+                            </relocations>
+
+                            <transformers>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+                                    <resource>META-INF/LICENSE-tomcat9-slf4j-logback</resource>
+                                    <file>${project.basedir}/LICENSE-tomcat9-slf4j-logback.txt</file>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+                                    <resource>META-INF/LICENSE-logback.txt</resource>
+                                    <file>${project.build.directory}/tomcat-slfj4-logback-config/assembly/LICENSE-logback.txt</file>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+                                    <resource>META-INF/LICENSE-slf4j.txt</resource>
+                                    <file>${project.build.directory}/tomcat-slfj4-logback-config/assembly/LICENSE-slf4j.txt</file>
+                                </transformer>
+                                <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
+                                    <resource>META-INF/LICENSE-tomcat.txt</resource>
+                                    <file>${project.build.directory}/tomcat-slfj4-logback-config/assembly/LICENSE-tomcat.txt</file>
```

Comment:
You seem to have these files checked in but not in this location, so I suspect that all these references are wrong.